### PR TITLE
Fix string serialization to tensor to allow null characters

### PR DESCRIPTION
**Pull request by CoderHam**
Fixes the TRTISTF_TensorSetString function to also use string length to include intermediate null characters.
PS: This does not have any test coverage Should I add the same.

### CODE REVIEW

File: src/backends/tensorflow/base_backend.cc [hunk 1]
```diff
@@ -609,7 +609,7 @@ BaseBackend::Context::SetStringInputTensor(
       content_byte_size -= len;
 
       TRTISTF_TensorSetString(
-          tensor, tensor_element_idx + element_idx, str.c_str());
+          tensor, tensor_element_idx + element_idx, str.c_str(), len);
```
Comment (deadeyegoodwin) [inline src/backends/tensorflow/base_backend.cc]: Don't need 'str' anymore, just use "content, len" here.
And move lines 608 and 609 to be just before 613

Comment (CoderHam) [reply src/backends/tensorflow/base_backend.cc]: Done

**Comment by deadeyegoodwin** [comment]
@CoderHam Can you file a ticket for 19.11 to add a test case for string containing a null character. We should be able to leverage some test we already have.

**Comment by CoderHam** [comment]
> @CoderHam Can you file a ticket for 19.11 to add a test case for string containing a null character. We should be able to leverage some test we already have.
Done
